automobile/sumo-interface.md

```diff
@@ -17,7 +17,9 @@ configuration files (.netccfg and .sumocfg).
 Then the interface will automatically start SUMO and run it in synchronization
 with Webots time. Each time a new vehicle enter the SUMO simulation, it will be
 created in Webots too and its position and orientation will be continually
-updated. If the simulation contains traffic lights, the name of the
+updated. The vehicle DEF name is set to `SUMO_VEHICLEX`, `X` been the vehicle number (starting from 0). If some vehicles whose DEF name is `SUMO_VEHICLEX` are already present in the world at the simulation start, then the interface will automatically use them before creating new vehicles, this can be usefull to avoid real-time addition of vehicles (wich can make the simulation speed drop for a very short time). If some vehicles whose DEF name is `WEBOTS_VEHICLEX` (X been the vehicle number starting from 0) are present in the simulation, the interface will automatically add them and update their position and orientation in SUMO in order to close the loop.
```

Comment:
I think you have to use ``being`` and not ``been`` in the sentence ``'X' been the vehicle number```.
And personally I would use ``with 'X' being ..``.

typo: usefull ->useful